bin/clonetb

```diff
@@ -41,6 +41,7 @@ usage() {
   echo "usage: $0 [-x] [--clone] [--ignore] [--unignore]"
   echo "  -x          clone cv32e40x subtree, known stable hash"
   echo "  --x-main    clone cv32e40x subtree, latest main branch"
+  echo "  --s-main    clone cv32e40s subtree, latest main branch"
```

Comment:
There should be an `-s` flag too!

But I want it to point to the merge-commit from https://github.com/openhwgroup/cv32e40s-dv/pull/3 after that PR is merged.